examples/config/config.docker.yaml

```
@@ -0,0 +1,36 @@
+# See detailed configuration settings : https://www.pomerium.com/docs/reference/
+
+
+# this is the domain the identity provider will callback after a user authenticates
+authenticate_service_url: https://authenticate.localhost.pomerium.io
+
+####################################################################################
+# Certificate settings:  https://www.pomerium.com/docs/reference/certificates.html #
+# The example below assumes a certificate and key file will be passed to the       #
+# Docker image.                                                                    #
+####################################################################################
+certificate_file: /pomerium/cert.pem
+certificate_key_file: /pomerium/privkey.pem
+
+##################################################################################
+# Identity provider settings : https://www.pomerium.com/docs/identity-providers/ #
+# The keys required in this section vary depending on your IdP. See the          #
+# appropriate docs for your IdP to configure Pomerium accordingly.               #
+##################################################################################
+idp_provider: google
+idp_client_id: REPLACE_ME
+idp_client_secret: REPLACE_ME
+
+# Generate 256 bit random keys  e.g. `head -c32 /dev/urandom | base64`
+cookie_secret: REPLACEMEWITHARANDOMKEY
+
+# https://pomerium.io/reference/#routes
+routes:
+  - from: https://verify.localhost.pomerium.io
+    to: http://verify
+    policy:
+      - allow:
+          or:
+            - email:
+                is: bdd@pomerium.io
```

```suggestion
                is: user@example.com
```
Could we globally replace all entries with my email?

---

> Could we globally replace all entries with my email?

I'm afraid not. Your email must live in infamy FOREVER!

... Yes, I'll do so in another branch.